File: class11/python/AP1101-CalculoSalario.py
# Calculo de salario liquido (estrutura sequencial)
bruto = float(input("Salario bruto: "))
inss = bruto * 0.11
ir = bruto * 0.075
liquido = bruto - inss - ir
print(f"INSS (11%): {inss:.2f}")
print(f"IR (7,5%): {ir:.2f}")
print(f"Salario liquido: {liquido:.2f}")

File: class11/java/AP1101-CalculoSalario.java
import java.util.Scanner;

public class CalculoSalario {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Salario bruto: ");
        double bruto = sc.nextDouble();
        double inss = bruto * 0.11;
        double ir = bruto * 0.075;
        double liquido = bruto - inss - ir;
        System.out.printf("INSS (11%%): %.2f%n", inss);
        System.out.printf("IR (7,5%%): %.2f%n", ir);
        System.out.printf("Salario liquido: %.2f%n", liquido);
    }
}
